protocols/mdns/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.43.1 [unreleased]
```

Comment:
This missed a version bump in `protocols/mdns/Cargo.toml`. I will make the bump with the upcoming release pull request.